ask_chatgpt.gemspec

```
@@ -13,13 +13,16 @@ Gem::Specification.new do |spec|
   spec.metadata["homepage_uri"] = spec.homepage
 
   spec.files = Dir.chdir(File.expand_path(__dir__)) do
-    Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
+    Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md", "bin/ask_chatgpt"]
   end
 
+  spec.executables   = ["ask_chatgpt"]
```

I think you can remove additional spaces here
```suggestion
  spec.executables = ["ask_chatgpt"]
```